src/styles/landing-page.less

```diff
@@ -36,6 +37,55 @@
     flex: 1 1 0%; // for .word-break()
     .word-break();
   }
+  .dropdown-menu {
+    margin-top: 0;
+    padding: 0;
+    width: 100%;
+    > li > a.catalog-search-match {
+      border-color: @color-pf-black-300;
+      border-width: 0 0 1px 0;
+      display: flex;
+      line-height: 1.5;
+      padding: 3px 10px 3px 5px;
+      white-space: normal;
+    }
+    > li > a.catalog-search-show-all {
+      background-color: @color-pf-black-200;
+      color: @link-color;
+      padding: 5px 0;
+      text-align: center;
+    }
+
+    // important needed to override styles declared important in PatternFly
+    .active > a.catalog-search-match, :focus{
+      background-color: @color-pf-blue-50 !important;
+      border-color: @color-pf-blue-100 !important;
+      color: inherit !important;
+    }
+    .active > a.catalog-search-show-all, :focus {
+      background-color: @color-pf-black-200 !important;
+      border-color: @color-pf-blue-100 !important;
+      color: @link-hover-color !important;
+    }
+
+    .catalog-search-show-none {
+      background-color: @color-pf-black-200;
+      border-color: @color-pf-black-300;
+      border-width: 0 0 1px 0;
+      padding: 5px 0;
+      text-align: center;
+    }
+  }
+  .search-pf-input-group {
+    .form-control{
+      font-size: (@font-size-base + 2);
```

Comment:
@font-size-base is currently 13px in console, so the result of this calculation is going to differ between the demo app and the console.  Double check this isn't problematic.

Reply:
In the PR to pull this into console, we over-ride this setting to get it back to 13px. Should we just set it to 13px here rather than using the base?

Reply:
Perhaps it would be easier just to set @font-size-base to 13px in the demo app?

Reply:
Good call.

Reply:
These settings were correct. The override in console is for having it in the masthead in project context.

Still think its worth updating the base font size in the catalog demo app.